docs/end-to-end-demo.md

```diff
@@ -1,8 +1,36 @@
 # End-to-End Demo
 In this guide, you will deploy Akri end-to-end, all the way from discovering local video cameras to the footage being streamed on a Web application. You will explore how Akri can dynamically discover devices, deploy brokers pods to perform some action on a device (in this case grabbing video frames and serving them over gRPC), and deploy broker services for obtaining the results of that action.
 
-**Note:** The first step can be performed either via K3s or MicroK8s. Select and
-carry out one or the other, then continue on with the rest of the steps. 
+## Set up mock udev video devices
```

Comment:
I'm not sure i understand the reordering. To me, it makes sense to start with a cluster.

Reply:
Oh i see what you mean.